controllers/landing.js

```diff
@@ -10,7 +10,8 @@ exports.get_landing= function(req,res,next){
 exports.submit_chatMessage= function(req,res,next){
     return models.Message.create({
         message: req.body.chat_message,
-        email: req.user.email
+        email: req.user.email,
```

Comment:
do we still need to pass `email` into this create? are we still tracking that attribute on `Message`?

Reply:
updated to remove email getting passed